api/database/seeders/PoolSeeder.php

```diff
@@ -38,7 +40,15 @@ public function run()
             $identifier = [
                 'name->en' => $pool['name']['en'],
             ];
-            Pool::updateOrCreate($identifier, $pool);
+            $newPool = Pool::updateOrCreate($identifier, $pool);
+            // Attach data to pool
+            $assets = CmoAsset::inRandomOrder()->limit(4)->get();
+            $classifications = Classification::inRandomOrder()->limit(5)->get();
+            $requirements = OperationalRequirement::inRandomOrder()->limit(5)->get();
+            $newPool->essentialCriteria()->saveMany($assets->slice(0,2));
+            $newPool->assetCriteria()->saveMany($assets->slice(2,2));
+            $newPool->classifications()->saveMany($classifications);
+            $newPool->operationalRequirements()->saveMany($requirements);
```

Comment:
This is going to add more and more criteria/classifications/requirements, each time seeders are run :/

Reply:
```
        foreach ($pools as $poolData) {
            $identifier = [
                'key' => $poolData['key'],
            ];
            $poolModel = Pool::where($identifier)->first();
            if (!$poolModel) {
                Pool::factory()->create($poolData);
            } else {
                $poolModel->update($poolData);
            }
        }
        ```

Reply:
🙏 thank you